src/main/java/jpo/sdw/depositor/depositors/SDWDepositor.java

```diff
@@ -29,7 +29,7 @@ public SDWDepositor(DepositorProperties props, JavaMailSender sender, WebClient
 
    @Override
    public void deposit(String message) {
-      Mono<ClientResponse> clientResponse = this.getWebClient().post().body(BodyInserters.fromObject(message))
+      Mono<ClientResponse> clientResponse = this.getWebClient().post().body(BodyInserters.fromValue(message))
             .exchange();
```

Comment:
".exchange()" looks to be deprecated at this point. Can we update that? Looks like maybe we can use ".retrieve()" as a decent replacement.

Reply:
@payneBrandon, I got that exchange call replaced and the unit tests adjusted